pkg/didcomm/protocol/mediator/service.go

```diff
@@ -748,6 +750,9 @@ func (s *Service) setKeyUpdateResponseCh(msgID string, keyUpdateCh chan *Keylist
 }
 
 func (s *Service) ensureConnectionExists(connID string) error {
+	//nolint:forbidigo
+	fmt.Println(fmt.Sprintf("~~~~ get router connID in routeStore: "+routeConnIDDataKey, connID))
```

Comment:
Can we remove this fmt statement ? If needed, we can use logger instead.

Reply:
removed